packages/subgraph/schema.graphql

```diff
@@ -462,152 +452,356 @@ type StandardCallback @entity {
   operation: PermissionOperation!
+
+  " The address of the target contract to grant `who` permission to. "
   where: Bytes!
+
+  " The address (EOA or contract) to which the permission has to be granted. "
   who: Bytes!
+
+  " The address of the contract containing the conditions which authorize a call by another address. This contract must inherit from `PermissionCondition` and implement the `IPermissionCondition` interface. "
   condition: Bytes
+
+  " Bytes32, the Keccak256 hash function of the ID for the permission. Normally the Keccak256 of the permission name (string). "
   permissionId: Bytes!
 }
 
+"""
+Wether the permission will be granted, granted with a condition or revoked in your DAO or plugin. (refer to `Objects/Permission` in this documentation site for more information about Aragon Permission system)
+"""
 enum PermissionOperation {
+  " A `Grant` permission operation, sets to ALLOW_FLAG the `permissionHashed` mapping in your DAO or plugin. "
   Grant
+
+  " A `Revoke` permission operation, sets to UNSET_FLAG the `permissionHashed` mapping in your DAO or plugin. "
   Revoke
+
+  " A `GrantWithCondition` permission operation, sets the `permissionHashed` mapping in your DAO or plugin to the contract address with the condition for this specific permission. "
   GrantWithCondition
 }
 
+"""
+The previous step to installing/uninstalling a plugin to/from your DAO. All `PluginSetup` SCs must implement the prepare functions containing
+necessary permissions and settings to install/uninstall a plugin.
+"""
 type PluginPreparation @entity(immutable: true) {
-  id: ID! # psp setupId + installationId
+  " Concatenation of DAO address, plugin address and the preparedSetupId. Joined by '_'. "
+  id: ID!
+
+  " The application of this preparation step. "
   installation: PluginInstallation!
+
+  " The caller of the preparation function (install/uninstall/update) in the `PluginSetupProcessor` SC. "
   creator: Bytes!
+
+  " The DAO where the plugin will be installed/uninstalled/updated. "
   dao: Dao!
-  preparedSetupId: Bytes! # the psp setupId
+
+  " The hash of the [preparedSetupId](https://devs.aragon.org/docs/osx/reference-guide/framework/plugin/setup/PluginSetupProcessorHelpers#internal-function-_getpreparedsetupid). "
+  preparedSetupId: Bytes!
+
+  " The `PluginRepo` contract for the plugin being prepared. "
   pluginRepo: PluginRepo!
+
+  " The version of the plugin being prepared. "
   pluginVersion: PluginVersion!
+
+  " The bytes-encoded data containing the input parameters for the installation as specified in the plugin's build metadata JSON file. "
   data: Bytes
+
+  " The deployed plugin SC by the `PluginSetup` SC. "
   pluginAddress: Bytes!
+
+  " The address array of helpers (contracts or EOAs) associated with this plugin version. "
   helpers: [Bytes!]!
+
+  " The array of multi-targeted permission operations to be applied by the `PluginSetupProcessor` to install/uninstall/update the plugin in the DAO. "
```

Comment:
I don't think you strictly need to apply multi target permissions

Reply:
We are inside PluginPreparation entity, which has permissions that will be handled by the `psp` which only uses `applyMultiTargetPermission` function